src/main/java/com/conveyal/taui/models/AnalysisRequest.java

```diff
@@ -86,9 +83,10 @@ public AnalysisTask populateTask (AnalysisTask task, Project project) {
             modifications = modificationsForProject(project.accessGroup, projectId, variantIndex);
         }
 
-        // Generate a checksum of all the modifications
+        // Generate a checksum of all the modifications and request data to uniquely identify this job for the broker
```

Comment:
It looks like we generate a checksum of the modifications themselves, intentionally _excluding_ the rest of the request data. The objective is not to uniquely identify the job, it is to provide a sort of semantic identifier for a version of the scenario that will in fact not change across multiple requests as long as the list of modifications has not changed.

Reply:
Updated to only use the modifications for the checksum